src/core/Cutter.cpp

```diff
@@ -4022,6 +4048,99 @@ QList<XrefDescription> CutterCore::getXRefs(RVA addr, bool to, bool whole_functi
     return xrefList;
 }
 
+void CutterCore::addGlobalVariable(RVA offset, QString name, QString typ)
+{
+    name = sanitizeStringForCommand(name);
+    CORE_LOCK();
+	char *errmsg = NULL;
+	RzType *globType = rz_type_parse_string_single(core->analysis->typedb->parser,
+			typ.toStdString().c_str(), &errmsg);
+	if (errmsg) {
+		qWarning() << tr("Error parsing type: \"") << typ << tr("\" message: ") << errmsg;
```

Comment:
```suggestion
		qWarning() << tr("Error parsing type: \"%1\" message: ").arg(typ) << errmsg;
```
same for all the others.